scripts/python/airflow/dags/contralory_declaration_download_pdfs.py

```diff
@@ -151,6 +154,12 @@ def download_pdf(remote_id: str,
     except NetworkError as ne:
         print(f"An error trying to get the file info {str(ne)}, downloading anyway")
 
+    numdata = keep_num_data(file_prefix)
+    if not is_valid_ci(numdata):
+        return None
```

Comment:
Add a warning message here. We should have a place to report all errors for a later review, maybe something like sentry.

Reply:
changes available in d88dd2a